CONTRIBUTING.md

```diff
@@ -87,69 +87,158 @@ You may need to install the dev_requirements for this
+    azext_iot_testrg=
+    azext_iot_testhub=
+    azext_iot_testdps=
+    azext_iot_teststorageaccount=
+    azext_iot_teststoragecontainer=
+    azext_iot_central_app_id=
+    azext_iot_central_scope_id=
+    azext_iot_central_primarykey=
+    azext_iot_central_storage_cstring=
+    azext_iot_central_storage_container=
+    azext_dt_adx_cluster=
+    azext_dt_adx_database=
+    azext_dt_adx_rg=
+    azext_dt_ep_eventgrid_topic=
+    azext_dt_ep_servicebus_namespace=
+    azext_dt_ep_servicebus_policy=
+    azext_dt_ep_servicebus_topic=
+    azext_dt_ep_eventhub_namespace=
+    azext_dt_ep_eventhub_policy=
+    azext_dt_ep_eventhub_topic=
+    azext_dt_ep_eventhub_topic_consumer_group=
+    azext_dt_ep_rg=
+    azext_dt_region=
 ```
 
-`azext_iot_teststorageaccount` is the storage account used for running iothub storage tests. Optional variable, specify only when you want to run storage tests. During these tests, your hub will be assigned a System-Assigned AAD identity, and will be granted the role of "Storage Blob Data Contributor" on the storage account you provide. Both the hub's identity and the RBAC role will be removed once the test completes.
+To run almost all of the tests, only the `azext_iot_testrg` is needed.
+
+To run all tests, `azext_iot_testrg`, `azext_iot_central_app_id`, and `azext_iot_central_primarykey` are needed because the IoT Central Primary Key cannot be currently retrieved through the CLI.
+
+For all resources, if the environmental variable is not provided, a new instance will be created for the test run and deleted at the end of the test run.
+
+##### General Variables
+
+`az_iot_testrg` is the resource group that contains the IoT Hub and DPS instances or where all test resources are created. This is required, as it will be the default resource group if any other resource group variables are not provided.
+
+`azext_iot_testhub` is the test IoT Hub name. Optional variable, used for IoT Hub and DPS tests. If not provided, a new IoT Hub will be created for the test run (and deleted at the end of the test).
+
+`azext_iot_testdps` is the test DPS name. Optional variable used for IoT DPS tests. If not provided, a new DPS instance will be created for the test run (and deleted at the end of the test).
+
+`azext_iot_teststorageaccount` is the storage account used for running IoT Hub and Central storage tests. Optional variable, specify only when you want to run storage tests. During these tests, your hub will be assigned a System-Assigned AAD identity, and will be granted the role of "Storage Blob Data Contributor" on the storage account you provide. Both the hub's identity and the RBAC role will be removed once the test completes. No role assignments are made for the IoT Central App.
 
 `azext_iot_teststoragecontainer` is the name of blob container belonging to the above mentioned storage account. Optional environment variable, defaults to 'devices' when not specified.
 
+##### IoT Central Test variables
```

Comment:
What about a markdown table to organize the vars for a sharper presentation? Columns could be  corresponding to the var  name, required and description.